openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -110,7 +110,7 @@ export function initBorrowAndReadLinks() {
     /* eslint-disable no-unused-vars */
     // used in openlibrary/macros/AvailabilityButton.html and openlibrary/macros/LoanStatus.html
     $(document).ready(function(){
-        $('#borrow_ebook,#read_ebook').on('click', function(){
+        $('.cta-btn--borrow,.cta-btn--read').on('click', function(){
```

Comment:
@cdrini this is existing code, but am wondering what it's for. It seems like borrow and read are always external links that trigger an HTTP request so this click handler never runs. Any ideas? Can it be removed?